resources/assets/js/label-trees/components/labelTreeLabel.vue

```diff
@@ -12,11 +12,9 @@
                 <span v-text="label.name" @click.stop="toggleSelect" @mouseenter="dontHover"></span>
             </span>
             <span class="label-tree-label__buttons">
-                <span class="label-tree-label_position">
-                    <span v-if="showFavouriteShortcuts" class="text-muted">
+                <span v-if="showFavouriteShortcuts" class="text-muted" :class="label-tree-label_position">
```

Comment:
This throws an error now. It's an easy fix that I could do myself but I'd like you to understand what happens here. The `:class` is a Vue thing that can set a class based on a variable or computed property of the Vue instance. However, the variable or computed property `label-tree-label_position` does not exist. It is just the static string of the class name. This is how you add multiple CSS classes to an element:
```suggestion
                <span v-if="showFavouriteShortcuts" class="text-muted label-tree-label_position">
```

Also please test your own UI changes with the developer console open before you request a review in the future :wink:

Reply:
Can you explain me how to test that or send me where it it described?

Reply:
You can test UI stuff of BIIGLE like this only by manually interacting with the UI in the browser. In this case, run `npm run watch` to make sure the compiled JS is up to date. Then open the image annotation tool and create a favourite label in the sidebar. Now press <kbd>F12</kbd> to open the browser developer tools and select the "Console" tab. You will see the JS errors there.

Reply:
Thanks!